crates/sui-test-transaction-builder/src/lib.rs

```diff
@@ -74,6 +74,30 @@ impl TestTransactionBuilder {
         )
     }
 
+    pub fn call_nft_create(self, package_id: ObjectID) -> Self {
+        self.move_call(
+            package_id,
+            "devnet_nft",
+            "mint",
+            vec![
+                CallArg::Pure(bcs::to_bytes("example_nft_name").unwrap()),
+                CallArg::Pure(bcs::to_bytes("example_nft_description").unwrap()),
+                CallArg::Pure(
+                    bcs::to_bytes("https://sui.io/_nuxt/img/sui-logo.8d3c44e.svg").unwrap(),
+                ),
+            ],
+        )
+    }
+
+    pub fn call_nft_delete(self, package_id: ObjectID, nft_to_delete: ObjectRef) -> Self {
```

Comment:
How often are these two called? Trying to see if they are builder-worthy

Reply:
Not too often. They too can be moved to the separate `devet_nft_transaction_utils` you suggested.